<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>361</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="credentials">
     <attribute name="title">
      <string>Credentials</string>
     </attribute>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>10</y>
        <width>441</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="categoryBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addItemButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="categoryEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addCategoryButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Login:</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>61</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Category:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteElementButton">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>290</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QLabel" name="warningLabel">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>297</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Encryption key missing! </string>
      </property>
     </widget>
     <widget class="QPushButton" name="learnMoreKeyMissingButton">
      <property name="geometry">
       <rect>
        <x>656</x>
        <y>290</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Learn more</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="securityAndDataAccess">
     <attribute name="title">
      <string>Security and Data Access</string>
     </attribute>
     <widget class="QLineEdit" name="generatePasswordEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="includeLettersBox">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Include letters</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="includeNumbersBox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Include numbers</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="includeSpecialCharactersBox">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Include special characters</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lengthPasswordEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="generateButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="includingUpperCaseLettersBox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Including upper case letters</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="excludeSpecificCharactersEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>188</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="auditPasswordBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordAuditEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="auditPasswordButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="encryptionKeyEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="encryptionKeyButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLabel" name="warningEncryptionKey">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>The file key.dat with your key is stored in the root folder.</string>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="generate_password_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lengthPasswordLabel">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Length:</string>
      </property>
     </widget>
     <widget class="QLabel" name="EcludeSpecificCharactersLabel">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Exclude specific characters:</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_audit_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password audit:</string>
      </property>
     </widget>
     <widget class="QLabel" name="encryption_key_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Encryption key:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="aboutText">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>150</y>
        <width>191</width>
        <height>131</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Rostyslav Nosikov / 2023&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;rostyslav.nosikov@outlook.com&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;GitHub:&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;X:&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-- PassLockBox  --&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
